pobieracz_danych_gugik.py

```diff
@@ -1616,9 +1555,36 @@ def osnowa_selected_btn_clicked(self):
         QgsApplication.taskManager().addTask(task)
         QgsMessageLog.logMessage('runtask')
 
-    # endregion
+    def mesh3d_fromLayer_btn_clicked(self):
+        connection = service_api.check_internet_connection()
+        if not connection:
+            self.show_no_connection_message()
+            return
+        path = self.dockwidget.folder_fileWidget.filePath()
+        if not self.checkSavePath(path):
+            return False
+        bledy = 0
+        layer = self.dockwidget.mesh3d_mapLayerComboBox.currentLayer()
+        if layer:
+            points = self.pointsFromVectorLayer(layer, density=1000)
```

Comment:
![image](https://github.com/envirosolutionspl/pobieracz_danych_gugik/assets/113895660/5fd85bb9-4822-4128-aefb-2af15a29f3d2)
1000 m to chyba będzie za dużo. Siatka rozmieszczenia modeli mesh ma 200x200

Reply:
byłabym za tym żeby to jakoś ujednolicić - bo w wielu miejscach to jest, może trzeba by pomyśleć nad jakimś innym sposobem szukania tych danych do pobrania